eng/common/templates/steps/generate-sbom.yml

```diff
@@ -5,9 +5,9 @@
 
 parameters:
   PackageVersion: 7.0.0
-  BuildDropPath: '$(Build.SourcesDirectory)/artifacts'
+  BuildDropPath: '$(Build.SourcesDirectory)\artifacts'
   PackageName: '.NET'
-  ManifestDirPath: $(Build.ArtifactStagingDirectory)/sbom
+  ManifestDirPath: '$(Build.SourcesDirectory)\artifacts'
```

Comment:
As the readme in the `eng/common/` directory says, these files can be auto-updated by automation.  NuGet.Client hasn't opt into Arcade SDK updates yet, but if we do, we will lose this customization.

Looking at [this file in the arcade repo](https://github.com/dotnet/arcade/blob/main/eng/common/templates/steps/generate-sbom.yml) we can see the contents are different, so as soon as we opt into arcade updates, we will lose this.

Reply:
Can you elaborate on the action you're proposing? We had a customization, we are not using arcade, if we do use arcade, we would not have had the customization we had nor will we have the one in this PR? Does that sum it up? 
How do we proceed?

Reply:
We discussed offline. Once we migrate to arcade, this will go away, and that's not a reason of concern for this PR.